addons/account/i18n/account.pot

```diff
@@ -13523,10 +13565,12 @@ msgid "The company this distribution line belongs to."
 msgstr ""
 
 #. module: account
+#. odoo-python
 #: code:addons/account/models/partner.py:0
 #, python-format
 msgid ""
-"The country of the foreign VAT number could not be detected. Please assign a country to the fiscal position or set a country group"
```

Comment:
looks like this was added to the wrong pot file somehow :facepalm: The string is in `base_vat`, but isn't in its pot file, https://github.com/odoo/odoo/pull/173976

Not sure if you want to re-export that one too

Reply:
It's there https://github.com/odoo/odoo/blob/17.0/addons/base_vat/i18n/base_vat.pot#L207

Reply:
Ah oops, sorry I missed that the strings are slightly different between the 2 modules